test/e2e_env/multizone/meshservice/status.go

```diff
@@ -14,6 +16,11 @@ import (
 )
 
 func MeshService() {
+	if Config.IPV6 {
+		fmt.Println("Test not supported on IPv6 yet")
+		return
+	}
```

Comment:
Yiou should use: `Label("ipv6-not-supported")` it's an easier way to track what tests don't run where.